freud/diffraction.py

```diff
@@ -508,8 +500,8 @@ def plot(self, ax=None, **kwargs):
         Returns:
             (:class:`matplotlib.axes.Axes`): Axis with the plot.
         """
-        if not _HAS_MPL:
-            raise ImportError(msg_mpl)
```

Comment:
To appease the pre-commit and the linter you want to revert the changes on the conditional imports to the code that is current with the main branch.